Integrations/src/main/java/io/deephaven/integrations/learn/Scatterer.java

```diff
@@ -47,10 +47,8 @@ public Object scatter(int idx, FutureOffset fo) {
         String[] queryStrings = new String[outputs.length];
 
         for (int i = 0; i < outputs.length; i++) {
-            // TODO: TICKET #1026: Replace typeString with the following once ticket #1009 is resolved:
             final String typeString = outputs[i].getType() == null ? "" : "(" + outputs[i].getType() + ")";
 
-            // final String typeString = "";
             queryStrings[i] = String.format("%s = %s (__scatterer.scatter(%d, %s))", outputs[i].getColName(),
                     typeString, i, futureOffsetColName);
```

Comment:
I think this is out of date with the fix I put into main.  I'm guessing you need a rebase.